### PR TITLE
CHANGES.md entries for 1.17.1

### PR DESCRIPTION
**Pull request by bradcray**

### CODE REVIEW

File: CHANGES.md [hunk 1]
```diff
@@ -1,6 +1,42 @@
...
+---------
+* fixed a bug in which postinit() was not called for generic records
+* fixed a bug in which most launchers would not work with 'make install'
+* fixed a portability problem w.r.t. our processing of /proc/cpuinfo
+* fixed a bug in which LinearAlgebra.sparse.eye() would unintentionally promote
+
+Cray-specific Changes
+---------------------
+* reduced contention for polling threads for CHPL_COMM=ugni
+* fixed a bug w/ unregisterable pre-existing memory segments for CHPL_COMM=ugni
```
Comment (gbtitus) [inline CHANGES.md]: I wouldn't characterize the old behavior as a "bug".  How about "improved how we handle unregisterable memory segments for CHPL_COMM=ugni"?  (It fits in 72 columns.)

Comment (bradcray) [reply CHANGES.md]: See if you like the rewording I just pushed (tried to avoid the first-person pronoun)

Comment (gbtitus) [reply CHANGES.md]: Yep, that's great.  I wondered about "we" but decided it was okay after finding some instances of "our".  Granted it's not quite the same, though.

**Comment by bradcray** [comment]
@ben-albrecht: I think I'm still waiting for your ack on your part of these changes.

**Comment by bradcray** [comment]
On second thought, I'm going to merge now and hope you can give me a post-merge shout if anything seems wrong.

File: CHANGES.md [hunk 2]
```diff
@@ -1,6 +1,42 @@
...
+
+Update to twentieth public release of Chapel, April 2018
+(see also changes below for 1.17.0)
+
+Bug Fixes
+---------
+* fixed a bug in which postinit() was not called for generic records
+* fixed a bug in which most launchers would not work with 'make install'
+* fixed a portability problem w.r.t. our processing of /proc/cpuinfo
+* fixed a bug in which LinearAlgebra.sparse.eye() would unintentionally promote
```
Comment (ben-albrecht) [inline CHANGES.md]: Technically, it's `LinearAlgebra.Sparse.eye()`

**Pull request merged**